modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtInstancelevel.testcase

```diff
@@ -0,0 +1,329 @@
+			ImportExport.unzipDownloadedExportFile();
+
+			var response = AccountAPI.getAccounts();
+
+			var dateCreated = JSONPathUtil.getProperty(
+				property = "$.items..dateCreated",
+				response = ${response});
+			var dateModified = JSONPathUtil.getProperty(
+				property = "$.items..dateModified",
+				response = ${response});
+			var externalReferenceCode = JSONPathUtil.getProperty(
+				property = "$.items..externalReferenceCode",
+				response = ${response});
+			var accountEntryId = JSONPathUtil.getProperty(
+				property = "$.items..id",
+				response = ${response});
+			var logoURL = JSONPathUtil.getProperty(
+				property = "$.items..logoURL",
+				response = ${response});
+			var name = JSONPathUtil.getProperty(
+				property = "$.items..name",
+				response = ${response});
+			var type = JSONPathUtil.getProperty(
+				property = "$.items..type",
+				response = ${response});
+			var fieldsValuesArray1 = "${dateCreated};dateCreated,${dateModified};dateModified,${externalReferenceCode};externalReferenceCode";
+			var fieldsValuesArray2 = "${accountEntryId};id,${logoURL};logoURL,${name};name,${type};type";
+
+			var totalFieldsValuesArray = "${fieldsValuesArray1},${fieldsValuesArray2}";
+
+			for (var i : list ${totalFieldsValuesArray}) {
+				var expectedValue = StringUtil.extractFirst(${i}, ";");
+				var fieldName = StringUtil.extractLast(${i}, ";");
+
+				BatchEngine.assertExportedFileContainsCorrectObject(
+					expectedValue = ${expectedValue},
+					fileName = "export.batch-engine-data.json",
+					jsonObject = "$.items..${fieldName}");
+			}
+
+			BatchEngine.assertExportedFileContainsCorrectObject(
+				expectedValue = "/o/headless-batch-engine/v1.0/import-task/com.liferay.headless.admin.user.dto.v1_0.Account",
+				fileName = "export.batch-engine-data.json",
+				jsonObject = "$.actions.createBatch.href");
+
+			BatchEngine.assertExportedFileContainsCorrectObject(
+				expectedValue = "com.liferay.headless.admin.user.dto.v1_0.Account",
+				fileName = "export.batch-engine-data.json",
+				jsonObject = "$.configuration.className");
+		}
+	}
+
+	@priority = 5
+	test CanSelectFromAllExistingFields {
+		task ("When selecting 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+
+			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+		}
+
+		task ("Then the list of fields to select contains all the fields available with GET request except the relationship nodes") {
+			var exportFieldsList1 = "name,actions,creator,dateCreated,dateModified,externalReferenceCode,id,keywords";
+			var exportFieldsList2 = "r_accountStudents_accountEntryERC,r_accountStudents_accountEntryId,scopeKey,status";
+
+			ImportExport.mapExport(exportFields = "${exportFieldsList1},${exportFieldsList2}");
```

Comment:
Same here

Reply:
I think it's better to split them in two or three sub list, if we use one sentence, it's too long and ugly

Reply:
What you shared to me is this is the previuys advice from Meg, could you share more details? Cause for me, I think even if it is long, sf will split into two or three lines, and we want to check the exact values, if there is an error/change in the future, we can compare the whole list with updated list easily.